### DISCUSSION TITLE
Weird Color Contrast Issue

### DISCUSSION THREAD

**Discussion by yonderbread** [qutebrowser/qutebrowser#6728] [Q&A] [answered]
[picture cuz i couldnt take a screenshot](https://media.discordapp.net/attachments/213077219424206858/894829401630445598/unknown.png)

Yeah.. so the color contrast on mouse hover tooltips looks like trash and I can't figure out how to change it. Any ideas?

**Comment by The-Compiler** [comment, marked as answer]
See #4520
